src/util.ts

```diff
@@ -8,7 +8,7 @@ export async function podsForNode(api: CoreV1Api, nodeName: string): Promise<V1P
 
 export function findSuffix(quantity: string): string {
     let ix = quantity.length - 1;
-    while (ix >= 0 && !/[\.0-9]/.test(quantity.charAt(ix))) {
+    while (ix >= 0 && !/[.0-9]/.test(quantity.charAt(ix))) {
```

Comment:
I think this is a breaking change we want the escaped `\.` to just match <dot> vs <dot> in regexp which matches all characters?

Reply:
That change was related to this error:

>   11:26  error  Unnecessary escape character: \.  no-useless-escape

Here are the docs for [no-useless-escape](https://eslint.org/docs/latest/rules/no-useless-escape). I can put it back and disable the lint rule for that line if you'd prefer.

Reply:
ok, thanks for checking.